config/navigation.rb

```diff
@@ -50,6 +50,7 @@
       s.item :dashboard, safe_join([fa_icon('tachometer fw'), t('admin.dashboard.title')]), admin_dashboard_path, if: -> { current_user.can?(:view_dashboard) }
       s.item :settings, safe_join([fa_icon('cogs fw'), t('admin.settings.title')]), admin_settings_path, if: -> { current_user.can?(:manage_settings) }, highlights_on: %r{/admin/settings}
       s.item :rules, safe_join([fa_icon('gavel fw'), t('admin.rules.title')]), admin_rules_path, highlights_on: %r{/admin/rules}, if: -> { current_user.can?(:manage_rules) }
+      s.item :warning_presets, safe_join([fa_icon('warning fw'), t('admin.warning_presets.title')]), admin_warning_presets_path, highlights_on: %r{/admin/warning_presets}, if: -> { current_user.can?(:manage_rules) }
```

Comment:
I think this should be `current_user.can?(:manage_settings)` since that's the underlying policy in https://github.com/mastodon/mastodon/blob/44b2ee3485ba0845e5910cefcb4b1e2f84f34470/app/policies/account_warning_policy.rb

I also think this might be better placed under Moderation instead of Administration? Tough to say though.

Reply:
Yeah I went back and forth on this too. I kind of tied it to the idea of warning presets being in line with rule violations which are also in the administration section (although maybe also better served as part of moderation)

I'll adjust the display to reflect the policy on the page, I just wasn't seeing what it was when I originally wrote it, thanks for investigating.